ctapipe/coordinates/pointing_corrections.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+
+
+class PointingCorrection:
+    """
+    Base class for pointing correction schemes
+    """
+
+    def get_camera_trans_matrix(self):
+        """
+        Get dummy transformation matrix, returns diagonal unit array
+
+        Returns
+        -------
+        matrix: affine translate matrix for camera corrections
+        """
+        return np.matrix(np.diagflat(np.ones(3)))
+
+
+class HESSStylePointingCorrection(PointingCorrection):
```

Comment:
I think this is fairly similar to how all telescopes do it, not specific to HESS, so perhaps rename this something like "StandardPointingCorrection". In the docs you should also put the formula, and specify the order of operations, since S x R x T is not the same transform as T x R x S.   E.g. write how one would transform a vector [alt,az] using this matrix (that also depends if these are column or row-ordered matrices).

Reply:
I would rather not use something like `HESSStyle` or `Standard` but something that actually describes what it does.